pkg/config/user_config.go

```diff
@@ -252,7 +252,7 @@ type PagingConfig struct {
 	// diff-so-fancy
 	// delta --dark --paging=never
 	// ydiff -p cat -s --wrap --width={{columnWidth}}
-	Pager PagerType `yaml:"pager" jsonschema:"minLength=1"`
+	Pager PagerType `yaml:"pager"`
```

Comment:
These changes and the ones below in the same file could go into a separate commit (before this one).

Reply:
Moved it out of this commit. Changed the history of the commits though. I think if I force push now the links you posted will break. I can add comments below yours with the updated hashes though.

Reply:
These days github is pretty good at retaining commits that are mentioned in comments. I think it's safe to force-push, I'm relatively sure that the links will continue to work. I'll mention the two commits again here just to be sure: 4a437b3f9455 (before), 9267b2eb7551 (after)